### ISSUE TITLE
3.0.2 - Can't run council - lua error

### ISSUE THREAD

**Issue by behub-torch** [evil-morfar/RCLootCouncil2#201]
Hello,

Everyone else in raid when in the leader position does not get this error. For some reason (even after a profile reset) trying to bring up the voting frame either via '/rc show' or starting a session I get this error and I am unable to actually run the council.

```
3x RCLootCouncil\core.lua:2045: attempt to call method 'GetClass' (a nil value)
[string "@RCLootCouncil\core.lua"]:2045: in function `GetUnitClassColoredName'
[string "@RCLootCouncil\Modules\sessionFrame.lua"]:157: in function `fnDoCellUpdate'
[string "@RCLootCouncil\Libs\lib-st\Core.lua"]:793: in function `Refresh'
[string "@RCLootCouncil\Libs\lib-st\Core.lua"]:340: in function `SortData'
[string "@RCLootCouncil\Libs\lib-st\Core.lua"]:592: in function `SetData'
[string "@RCLootCouncil\Modules\sessionFrame.lua"]:78: in function `Show'
[string "@RCLootCouncil\ml_core.lua"]:1227: in function `Test'
[string "@RCLootCouncil\core.lua"]:726: in function `Test'
[string "@RCLootCouncil\Modules\options-Options.lua"]:219: in function <RCLootCouncil\Modules\options.lua:217>
[string "=[C]"]: ?
[string "@TomTom\libs\AceConfig-3.0-3\AceConfigDialog-3.0\AceConfigDialog-3.0-79.lua"]:51: in function <...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:49>
[string "@TomTom\libs\AceConfig-3.0-3\AceConfigDialog-3.0\AceConfigDialog-3.0-79.lua"]:843: in function <...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:664>
[string "=[C]"]: ?
[string "@TomTom\libs\AceGUI-3.0\AceGUI-3.0-41.lua"]:72: in function <TomTom\libs\AceGUI-3.0\AceGUI-3.0.lua:70>
[string "@TomTom\libs\AceGUI-3.0\AceGUI-3.0-41.lua"]:306: in function `Fire'
[string "@TomTom\libs\AceGUI-3.0-41\widgets\AceGUIWidget-Button.lua"]:22: in function <...mTom\libs\AceGUI-3.0\widgets\AceGUIWidget-Button.lua:19>
```

Also (related?) when I try to sync settings with another office it says the player doesn't exist and adds their server to the name multiple times so looks like ...

[08:50] No player named 'Joshyboi-Frostmourne-Frostmourne-Frostmourne' is currently playing.

**Comment by behub-torch** [comment]
[RCLootCouncil.zip](https://github.com/evil-morfar/RCLootCouncil2/files/5702171/RCLootCouncil.zip)

I just installed the beta hoping there was a fix in there.

**Comment by behub-torch** [comment]
`/rc clearcache` did not work as suggested in Discord for another user

**Comment by behub-torch** [comment]
Beta appears to work.

**Comment by evil-morfar** [comment]
Thanks for your report - should be fixed in v3.1.0
